configuration/cli/envmod/envmod.cpp

```diff
@@ -48,8 +48,8 @@ std::vector<std::string> splitString(const std::string &input, const std::string
 // Default configuration directories
 EnvironmentType envType = XML;
 std::string masterSchemaFile = "environment.xsd";
-std::string configSchemaDir = "";
-std::string configSchemaRelativeDir = COMPONENTFILES_DIR PATHSEPSTR "configschema" PATHSEPSTR "xsd" PATHSEPSTR;
+//std::string configSchemaDir = "";
+std::string configSchemaDir = COMPONENTFILES_DIR PATHSEPSTR "configschema" PATHSEPSTR "xsd" PATHSEPSTR;
```

Comment:
this seems to only be used in the usage output, is it actually needed?

Reply:
configSchemaDir seems to only be referenced in the usage, and doesn't seem to be reassigned, it should at least be const?

Reply:
configSchemaDir is settable with a parameter, so can't be const.